networking_dvs/plugins/ml2/drivers/mech_dvs/agent/vcenter_util.py

```diff
@@ -273,6 +284,7 @@ def stop(self):
     def _run(self):
         LOG.info(_LI("Monitor running... "))
 
+        self.vc_api = VCenter(self.config, pool=self.pool)
```

Comment:
I think, that might be causing trouble.
The VCenterMonitor is created by a VCenter itself here: https://github.com/mmidolesov/networking-dvs/blob/2fe1313aa229d074ee10ff9284f086a121b51771/networking_dvs/plugins/ml2/drivers/mech_dvs/agent/vcenter_util.py#L550

Could you pass the vc_api object instead?